### DISCUSSION TITLE
Anyway to unmount a tile by moving it outside of the ridable tile?

### DISCUSSION THREAD

**Discussion by demotech89** [Saibot393/Rideable#73] [Q&A]
Currently when a token mounts a tile and moves outside of the ridable tile, the tile moves with the token. 

Is there anyway to unmount the token by moving it outside of the tiles location? I am trying to make a car seat where a player is mounted in the car but does not need to hit "N" to unmount. Also the ridable button on the token does not seem to work for me for some reason.

**Comment by Saibot393** [comment]
So to unmount when leaving the the tile the repective player has to set the rideaböe setting "Rider movement" to "Unmount Rider" (or you can set the default setting "Rider movement default setting" to that).
Regarding the hud mounting button, did you have both the rider and the mount selected? Using the button while only having selected the mount only works for players, in which case the default character will be mounted/unmounted (assuming that is set)

**Reply by demotech89** [reply]
Thank you so much for responding.....I honestly thought I changed that to it....but apparently I did not....that totally worked. Also yeah, I was not selecting both for the button. Thanks for that as well.
